automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/methodconfigs/ImportMethodConfigModal.scala

```diff
@@ -0,0 +1,74 @@
+package org.broadinstitute.dsde.firecloud.page.workspaces.methodconfigs
+
+import org.broadinstitute.dsde.firecloud.FireCloudView
+import org.broadinstitute.dsde.firecloud.component._
+import org.broadinstitute.dsde.firecloud.component.Component._
+import org.broadinstitute.dsde.firecloud.page.{MethodTable, Modal}
+import org.openqa.selenium.WebDriver
+
+class ImportMethodConfigModal(implicit webDriver: WebDriver) extends Modal {
+  private val chooseConfigFromRepoModalButton = Button("import-from-repo-button")
+  // private val chooseConfigFromWorkspaceModalButton = Button("copy-from-workspace-button")
+
+  def chooseConfigFromRepo(methodNamespace: String, methodName: String, snapshotId: Int, methodConfigName: String, rootEntityType: Option[String]): Unit = {
+    chooseConfigFromRepoModalButton.doClick()
+    val importView = await ready new ImportFromMethodRepoView()
+    importView.select(methodNamespace, methodName, snapshotId, methodConfigName, rootEntityType)
+  }
+}
+
+private class ImportFromMethodRepoView(implicit webDriver: WebDriver) extends FireCloudView {
+  private val methodRepoTable = new MethodTable[MethodSummaryView]() {
+    override protected def awaitInnerPage(namespace: String, name: String): MethodSummaryView = {
+      await ready new MethodSummaryView
+    }
+  }
+
+  override def awaitReady(): Unit = methodRepoTable.awaitReady()
+
+  def select(methodNamespace: String, methodName: String, snapshotId: Int, methodConfigName: String, rootEntityType: Option[String]): Unit = {
+    methodRepoTable
+      .enterMethod(methodNamespace, methodName)
+      .confirm()
+      .selectConfiguration(methodNamespace, methodConfigName, snapshotId) // seems like the wrong way to supply namespace?
+      .confirm()
```

Comment:
yeah, I wonder about the config namespace here. the config's namespace is not necessarily the same as the method's namespace.